packages/opentelemetry-plugin-winston/src/utils.ts

```diff
@@ -0,0 +1,64 @@
+/*!
+ * Copyright 2019, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { TraceInformation, TRACE_PARAM_NAME, WinstonChunk } from './types';
+import * as types from '@opentelemetry/types';
+
+/**
+ * Get trace information from span
+ * @param span
+ */
+function getTraceInformation(span: types.Span): TraceInformation {
+  const traceInformation: TraceInformation = {
+    trace_id: span.context().traceId,
+    span_id: span.context().spanId,
+  };
+  return traceInformation;
+}
+
+/**
+ * Adds information to chunk about trace
+ * @param chunk
+ * @param span
+ */
+export function addTraceToChunk(
+  chunk: WinstonChunk,
+  span: types.Span
+): WinstonChunk {
+  const newChunk: WinstonChunk = Object.assign({}, chunk);
+  newChunk[TRACE_PARAM_NAME] = getTraceInformation(span);
+  return newChunk;
+}
+
+/**
+ * Process arguments and adds additional information about trace
+ * @param args
+ * @param span
+ */
+export function processArgs(args: any[], span: types.Span): any[] {
+  for (let i = args.length - 1; i >= 0; i--) {
+    if (typeof args[i] === 'object') {
+      args[i] = addTraceToChunk(args[i], span);
```

Comment:
Is there some reason we're modifying an existing chunk rather than adding our own?

Reply:
this param is optional if user already added some extra information then we have to add trace information into this object